### ISSUE TITLE
limit updates

### ISSUE THREAD

**Issue by tube0013** [briis/smartweatherudp#15]
I just noticed that the 3 second update rate for these sensor is filling up my db quite a bit.  If there is any movement toward a re-write of the component please consider limiting the frequency of the updates to HA.

**Comment by LuckyJayTx** [comment]
I am working on an overhaul to address several items, one of which is the 3 second update rate on all the sensors.  There are only a couple of them that need to update at that rate.

Can't say how long it will take, but it is being worked on...

**Comment by Awebian** [comment]
Or is there a way to set a rolling log size say 24 hours or 1 Week... and older data would just fall off / be removed.  New to HA. I like a solution that would set a max recorded data size say in definable hours by user vs a solution that limits the sample rate and would eventually fill the database just a a slower rate.  Might also be nice to have an option to flush the database and start over.   It’s easy to ask for all of this, but I know it can be difficult to implement.  Thank you all who contribute and work hard on the software solutions...

**Comment by tube0013** [comment]
@Awebian check the recorder integration documentation. The options to do this are there

**Comment by tube0013** [comment]
figured out a work-around for now... My states table was up to 30 gb and events 70gb with a huge chunk of the data from my smartwearther sensors. I was nervous about the 0.112 db schema update taking ages, so was looking for a solution. I ended up setting up filter sensors for all the smartweaterudp sensors with a throttle_time of 5 minutes. I dropped my old ha database and started anew with recorder using the new filtered sensors.

the filter sensors look like this:

```
sensor:
  - platform: filter
    name: "SmartWeather Temperature"
    entity_id: sensor.smartweatherudp_temperature
    filters:
      - filter: time_throttle
        window_size: "00:05"
```
